docs/manual/troubleshooting.tex

```diff
@@ -947,18 +947,32 @@ \subsection{Configure Eclipse to edit the Checker Framework\label{building-eclip
 \begin{enumerate}
 \item Follow the instructions earlier in Section~\ref{build-source} to
 clone and build all projects from their sources.
+\item Follow the instructions in the \<README-eclipse> document found in the
+\<\$CHECKERFRAMEWORK/../annotation-tools/annotation-file-utilities/lib>
+directory to download JDK 8 source code and create symbolic links to
+\<tools.jar> and \<src.zip>.
 \item Download Eclipse from
 the \href{https://www.eclipse.org/downloads/}{official Eclipse website}
 and install it.
-\item Run Eclipse.
-\item Enter the main Eclipse working screen and in the ``File'' menu,
-select ``Import'' $\rightarrow$ ``General'' $\rightarrow$ ``Existing
-Projects into Workspace''.
+\item Run Eclipse and enter the main Eclipse working screen.
+\item Checker Framework's sources depend on classes packaged in \<rt.jar> from
+the JRE. Attaching the source code to \<rt.jar> makes it easier to understand
+and work with these classes:
+In the ``Window'' menu, select ``Preferences'' $\rightarrow$ ``Java''
+$\rightarrow$ ``Installed JREs''. Select the Java 8 JRE, then Select ``Edit''.
+In the list of JRE system libraries, select \<rt.jar>. Select
+``Source Attachment'' $\rightarrow$ ``External location'' $\rightarrow$
+``External File''. Locate the directory pointed to by \<\$JAVA\_HOME>, select
+\<src.zip> and then OK to attach the source. Select ``Finish'' and
+``Apply and Close''.
+\item In the ``File'' menu, select ``Import'' $\rightarrow$ ``General''
+$\rightarrow$ ``Existing Projects into Workspace''.
 \item After the ``Import Projects'' window appears, select ``Select
 Root Directory'', and select the parent of the \code{\$CHECKERFRAMEWORK} directory.
 \item Ensure ``Search for nested projects'' is selected in the Options panel.
-\item Select all the projects in the folder except for ``personalblog-demo'',
-then select ``Finish'' to import the projects.
+\item Select all the projects in the list except for ``personalblog-demo''
+and ``jsr308-langtools'', then select ``Finish'' to import the projects.
```

Comment:
We no longer clone jsr308-langtools. Why would it now show up in this list of projects?

Reply:
`./gradlew cloneAndBuildDependencies` runs './.travis-build-without-test.sh', and in that script we download and extract jsr308-langtools: https://github.com/typetools/checker-framework/blob/master/.travis-build-without-test.sh#L65

@smillst please remove the .project and .classpath files in the zip we distribute for jsr308-langtools, then it won't show up in the import here

Reply:
If we already have a `jsr308-langtools` version cloned by `./gradlew cloneAndBuildDependencies` why not use it?
Why add the steps with `tools.jar` and `src.zip` if we have a clone of all of langtools?

Reply:
My understanding is that we don't want to clone `jsr308-langtools` anymore. Is this still true?